src/modules/AdminModule.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 #include "ProtobufModule.h"
-#if HAS_WIFI
+#if HAS_WIFI && !MESHTASTIC_EXCLUDE_WIFI
```

Comment:
Is this needed if MESHTASTIC_EXCLUDE_WIFI defines HAS_WIFI as 0?

Reply:
Oh, yes you’re right. That is redundant. Not at the computer now, but I can push the change later.

Reply:
Looking at this again, I think the original is correct. If MESHTASTIC_EXCLUDE_WIFI is not defined, HAS_WIFI can still be 0 or 1 (hardware dependent). Re-ordering the test might make the logic a bit more clear though.